src/main/java/com/mraof/minestuck/data/recipe/MinestuckCombinationsProvider.java

```diff
@@ -182,6 +232,19 @@ protected void buildCraftingRecipes(Consumer<FinishedRecipe> consumer)
 		CombinationRecipeBuilder.of(Items.VINE).input(ItemTags.LEAVES).and().input(Items.LADDER).buildFor(consumer, Minestuck.MOD_ID);
 		CombinationRecipeBuilder.of(Items.ROSE_BUSH).input(Items.LARGE_FERN).or().input(Items.POPPY).buildFor(consumer, Minestuck.MOD_ID);
 		CombinationRecipeBuilder.of(Items.SUGAR_CANE).input(Items.BAMBOO).and().input(Items.SUGAR).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.AZALEA).input(Items.SPYGLASS).or().input(Items.OAK_SAPLING).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.AZALEA_LEAVES).input(Items.AZALEA).and().input(Items.OAK_LEAVES).buildFor(consumer, Minestuck.MOD_ID);
```

Comment:
Leaves, saplings, logs, planks, etc should all be handled in the array started on line 36 unless we consider it an exception. Feel free to argue whether you think it constitutes an exception

Reply:
This was marked as resolved but I dont see an argument for keeping it here and it hasnt alternatively been moved

Reply:
Will be handled in PR to KingBalDoro:1.18-items